slam-libraries/viam-cartographer/README.md

```diff
@@ -5,9 +5,6 @@
 > **Warning**
 > This is an experimental feature. Stability is not guaranteed. Breaking changes are likely to occur, and occur often.
 
-> **Disclaimer**
-> Cartographer has not yet been integrated into RDK.
```

Comment:
[minor] `Cartographer has not yet been officially integrated into RDK.`

Reply:
There are two reasons why I can't entirely agree with making this change:
1. Cartographer is integrated into RDK since the user can follow the instructions to install cartographer and run it through RDK.
2. We're currently wrapping up the last few tickets to finish cartographer, and this task is one of them. If we were to make the change you suggested, we'd have to create another ticket for undoing it later again. I don't think that's worth the effort.